docs/dev.rst

```diff
@@ -428,107 +552,43 @@ this code and it has to be tested manually. You can do something like this:
-That superuser can promote other users to superusers too.
-
-This action does NOT require that the user signs in at least once. If the user
-does not exist, it gets created.
-
-The easiest way to create your first superuser is to use ``docker compose``:
-
-.. code-block:: shell
-
-   $ docker compose run --rm web bash python manage.py superuser yourname@example.com
-
-Additionally, in a local development environment, you can create a
-corresponding user in the oidcprovider service like this:
-
-.. code-block:: shell
-
-   $ docker compose exec oidcprovider /code/manage.py createuser yourname yourpassword yourname@example.com
-
-
-Giving users permission to upload symbols
------------------------------------------
-
-The user should write up a bug. See :ref:`upload-basics`.
-
-If the user is a Mozilla employee, needinfo the user's manager and verify the
-user needs upload permission.
+System tests
+------------
 
-If the user is not a Mozilla employee, find someone to vouch for the user.
+System tests are located in the repository in ``systemtests/``. See the
+``README.rst`` there for usage.
 
-Once vouched:
+System tests can be run against any running environment:
 
-1. Log in to `<https://symbols.mozilla.org/users>`_
-2. Use the search filter at the bottom of the page to find the user
-3. Click to edit and make give them the "Uploaders" group (only).
-4. Respond and say that they now have permission and should be able to either
-   upload via the web or create an API Token with the "Upload Symbol Files"
-   permission.
-5. Resolve the bug.
+* local: local dev environment
+* stage: the stage server environment
+* prod: the prod server environment--will not run destructive tests
 
+System tests can help verify that upload API and download API work. They
+periodically need to be updated as symbols files expire out of the systems.
 
-Viewing all metrics keys
-------------------------
 
-In the Symbols Service webapp, to get insight into all metrics keys that are
-used, a special Markus backend is enabled called
-``tecken.libmarkus.LogAllMetricsKeys``. It's enabled by default in local
-development. And to inspect its content you can either open
-``all-metrics-keys.json`` directly (it's git ignored) or you can run:
+Load tests
+----------
 
-.. code-block:: shell
+At various points, we've done some load testing of the system. The scripts are
+located in:
 
-   $ make shell
-   app@xxx:/app$ ./bin/list-all-metrics-keys.py
+https://github.com/mozilla-services/tecken-loadtests/
 
-Now you can see a list of all keys that are used. Take this and, for example,
-make sure you make a graph in Datadog of each and everyone. If there's a key in
-there that you know you don't need or care about in Datadog, then delete it
-from the code.
+They're generally unmaintained, but can be a good starting point for a new load
+testing effort.
 
-The file ``all-metrics-keys.json`` can be deleted any time and it will be
-recreated again.
 
 
-Localstack (S3 mock server)
```

Comment:
We used to use minio and I think this worked with minio, but it doesn't work with localstack, so I removed this section.